indico/modules/events/timetable/templates/pdf/_program.html

```diff
@@ -0,0 +1,340 @@
+    </div>
+{% endmacro %}
+
+{% macro render_block_date(entry, day, print_date_close_to_sessions) %}
+    {% if print_date_close_to_sessions %}
+        <p class="session-block-date">
+            <span class="bold">{{ day|format_skeleton('EEEEdMMMM', timezone=entry.event.timezone) }}</span>
+            <span class="timebox">
+                {{ format_interval(entry.start_dt.astimezone(entry.event.tzinfo), entry.end_dt.astimezone(entry.event.tzinfo), format='HHmm') }}
+            </span>
+        </p>
+    {% endif %}
+{% endmacro %}
+
+{% macro render_time(entry) %}
+    <div class="time">
+        <div class="time-from">
+            {{ entry.start_dt|format_time(timezone=entry.event.timezone) }}
+        </div>
+        <div class="time-divider">
+            {# Line between generated in CSS #}
+        </div>
+        <div class="time-divider">
+            {# Line between generated in CSS #}
+        </div>
+        <div class="time-to">
+            {{ entry.end_dt|format_time(timezone=entry.event.timezone) }}
+        </div>
+    </div>
+{% endmacro %}
+
+{# Utilities #}
+
+{% macro render_place(entry) %}
+    {% set entry = entry.object if entry.object else entry %}
+
+    {{ [entry.venue_name, entry.room_name, entry.address] | select | join(', ') }}
+{% endmacro %}
+
+{% macro render_names(people, show_title=false, show_affiliation=false) %}
+    {% for person in people %}
+        {%- if show_title %}
+            {{ person.title }}
+        {%- endif %}
+        {{ person.display_full_name }}
+        {%- if show_affiliation and person.affiliation %}
+            ({{ person.affiliation }})
+        {%- endif -%}
+        {%- if not loop.last %},{% endif %}
+    {% endfor %}
+{% endmacro %}
+
+{% macro get_custom_id(entry) -%}
+    {{ 'c' ~ entry.type ~ '-' ~ entry.id }}
+{% endmacro %}
+
+{% macro get_color_styles(entry) %}
+    {% set c_id = get_custom_id(entry) %}
+    {% set colors = entry.object.colors or entry.object.session.colors %}
+
+    <style>
+        .{{ c_id }} .inherited-bg-color {
+            background-color: #{{ colors.background }};
+        }
+
+        .{{ c_id }} .inherited-bg-color-trans {
+            background-color: #{{ colors.background }}66;
+        }
+
+        .{{ c_id }} .inherited-txt-color {
+            color: #{{ colors.text }};
+        }
+
+        .{{ c_id }} .inherited-border-color {
+            border-color: #{{ colors.background }};
+        }
+    </style>
+{% endmacro %}
+
+
+{% macro render_description(description) %}
+    <div class="session-block-content-box description inherited-border-color">
+        <ul class="contrib-timeline">
+            <li>
+                <h4>{% trans %}Description{% endtrans %}</h4>
+                {{ description }}
+            </li>
+        </ul>
+    </div>
+{% endmacro %}
+
+{% macro render_block_title(block) %}
+    <h3>
+        {{ block.session.title }}
+        {% if block.title %}: {{ block.title }}{% endif %}
+    </h3>
+{% endmacro %}
```

Comment:
`<h3>{{ block.full_title }}</h3>` - maybe no longer worth having a macro for it? Especially since in the one place where it's being used, there's already an `<h3>` around it, so you have two of them right now...

Reply:
Yep that's true and actually, this whole macro ended up being unnecessary now. I just do `block.full_title or block.title` and put that directly under the single `<h3>` where we use it. So saved some lines there for sure.